src/main/java/com/google/api/codegen/viewmodel/StaticLangXCombinedSurfaceView.java

```diff
@@ -0,0 +1,130 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.auto.value.AutoValue;
+
+import java.util.Collection;
+import java.util.List;
+
+@AutoValue
+public abstract class StaticLangXCombinedSurfaceView implements ViewModel {
+  @Override
+  public abstract String templateFileName();
+
+  @Override
+  public abstract String outputPath();
+
+  @Override
+  public String resourceRoot() {
+    return SnippetSetRunner.SNIPPET_RESOURCE_ROOT;
+  }
+
+  public abstract Integer servicePort();
+
+  public abstract List<String> imports();
+
+  public abstract Iterable<String> authScopes();
+
+  public abstract List<PathTemplateView> pathTemplates();
+
+  public abstract List<PathTemplateGetterFunctionView> pathTemplateGetters();
+
+  public abstract Collection<RetryPairDefinitionView> retryPairDefinitions();
```

Comment:
Use List instead of Collection to provide more consistency. I want to give the promise of ordering.